wikipron.py

```diff
@@ -153,7 +153,7 @@ def _get_li_selector(
             )
 
         if not dialect:
-            dialect_selector = "true"
+            dialect_selector = "true()"
```

Comment:
If I didn't make this change, grabbing all dialects (= not specifying any dialect, e.g., getting all pronunciations for "mocha" regardless of dialect in the test case) wouldn't work. Not sure why it didn't simply break the XPath parsing before, though.